tests/lang/core.pk

```diff
@@ -50,6 +59,17 @@ for i in 0..1000
   assert(abs(sin(i) / cos(i) - tan(i)) < threshold)
 end
 
+assert(abs(cosh(PI/2) - 2.5) < 0.01 )
+assert(abs(tanh(PI/4) - 1.0) < 0.5 )
+for i in 0..100
+  assert(abs(sinh(i) / cosh(i) - tanh(i)) <= 0.01)
+end
+
+assert(abs(acos(PI/4) - 1) < 0.35)
+assert(abs(atan(PI/4) - 0.5) < 0.2)
+for i in -1..1
```

Comment:
This is in the range of `[-1, 0]`, 
```ruby
x = -1, interval = 1000
for i in 0..interval
  x += 2/interval
  # use x in asin(x), acos(x)
end
```
And I totally forget about the `asin` and `acos` range (-1 to +1), which you have to handle as well.

Reply:
So I'll put -1..2, yes I forgot the to is exclusive. Thanks for the reminder.

Reply:
> So I'll put -1..2

No, it'll be just -1, 0 and 1 but we need all the numbers between -1 to +1 (all the floats). Use the code I've written above.

Reply:
Yes, you're correct. Thank you! We need all those floats in between, true!